R/IDateTime.R

```diff
@@ -356,7 +363,14 @@ isoweek = function(x) {
   1L + (nearest_thurs - year_start) %/% 7L
 }
 
-month   = function(x) convertDate(as.IDate(x), "month")
+month <- function(x, label = FALSE) {
+    dates <- as.IDate(x)
+    if (isTRUE(label)) {
+        return(format(dates, "%B"))
+    }
```

Comment:
Do we need to convert to `IDate` for using `format`?

Reply:
All of this family of functions use `as.IDate()` already, so need to note anything new here IMO.